lib/Config.js

```diff
@@ -155,41 +172,51 @@ class Config {
-                    this.utapi.redis.name = config.utapi.redis.name;
+                assert(typeof config.utapi.redis.name === 'string',
+                    'bad config: redis sentinel name must be a string');
+                this.utapi.redis.name = config.utapi.redis.name;
 
-                    assert(Array.isArray(config.utapi.redis.sentinels),
-                        'bad config: redis sentinels must be an array');
-                    config.utapi.redis.sentinels.forEach(item => {
-                        const { host, port } = item;
-                        assert(typeof host === 'string',
-                            'bad config: redis sentinel host must be a string');
-                        assert(typeof port === 'number',
-                            'bad config: redis sentinel port must be a number');
-                        this.utapi.redis.sentinels.push({ host, port });
-                    });
-                } else {
-                    // check for standalone configuration
-                    this.utapi.redis = {};
-                    assert(typeof config.utapi.redis.host === 'string',
-                        'bad config: redis.host must be a string');
-                    assert(typeof config.utapi.redis.port === 'number',
-                        'bad config: redis.port must be a number');
-                    this.utapi.redis.host = config.utapi.redis.host;
-                    this.utapi.redis.port = config.utapi.redis.port;
-                }
+                assert(Array.isArray(config.utapi.redis.sentinels),
+                    'bad config: redis sentinels must be an array');
+                config.utapi.redis.sentinels.forEach(item => {
+                    const { host, port } = item;
+                    assert(typeof host === 'string',
+                        'bad config: redis sentinel host must be a string');
+                    assert(typeof port === 'number',
+                        'bad config: redis sentinel port must be a number');
+                    this.utapi.redis.sentinels.push({ host, port });
+                });
+            } else {
+                // check for standalone configuration
+                this.utapi.redis = {};
+                assert(typeof config.utapi.redis.host === 'string',
+                    'bad config: redis.host must be a string');
+                assert(typeof config.utapi.redis.port === 'number',
+                    'bad config: redis.port must be a number');
+                this.utapi.redis.host = config.utapi.redis.host;
+                this.utapi.redis.port = config.utapi.redis.port;
             }
             if (config.utapi.metrics) {
                 this.utapi.metrics = config.utapi.metrics;
             }
             if (config.utapi.component) {
                 this.utapi.component = config.utapi.component;
             }
+            if (config.utapi.replaySchedule) {
```

Comment:
Add a comment here describing that the value is crontab style with an example. Also add `(optional)` to the comment suggesting that this is an optional config.